cmd/template.go

```diff
@@ -154,17 +154,20 @@ Errors:
 
 %3d - The specified launch template was not found.`,
 		NoSuchTemplate),
-	Args: cobra.ExactArgs(1),
+	Args: cobra.ExactArgs(2),
 	Run: func(_ *cobra.Command, args []string) {
-		omat := loadOmatConfig()
+		accountName := args[0]
+		templateName := args[1]
 
-		deployAcctDetails := awsutil.FindAndAssumeAdminRole(omat.DeployAccountSlug, omat)
+		omat := loadOmatConfig(accountName)
+
+		deployAcctDetails := awsutil.FindAndAssumeAdminRole(omat)
 		deployAcctEC2Client := ec2.New(deployAcctDetails.Session, deployAcctDetails.Config)
 
-		buildAcctDetails := awsutil.FindAndAssumeAdminRole(omat.BuildAccountSlug, omat)
+		buildAcctDetails := awsutil.FindAndAssumeAdminRole(omat)
```

Comment:
So this seems wrong -- this command needs to take a ci/cd account.

Though, I'm not sure what this command does. If you're getting AMIs out of something, the AMI will tell you what account it came from, and the account can be thunked to through `/omat/account_id_registry/<account_id>`

Reply:
f/u added to roadmap.